index.src.html

```diff
@@ -653,36 +656,14 @@
         1. Let |timestamp| and |duration| be the
             {{EncodedVideoChunk/timestamp}} and {{EncodedVideoChunk/duration}}
             from the {{EncodedVideoChunk}} associated with |output|.
-        2. Let |frame| be the result of running the [=Create a VideoFrame=]
-            algorithm with |output|, |timestamp|, and |duration|.
-        3. Invoke {{VideoDecoder/[[output callback]]}} with |frame|.
-  </dd>
-  <dt>
-    <dfn>Create a VideoFrame</dfn> (with |output|, |timestamp|, and |duration|)
-  </dt>
-  <dd>
-    1. Let |planes| be a sequence of {{Plane}}s containing the decoded
-        video frame data from |output|.
-    2. Let |pixelFormat| be the {{PixelFormat}} of |planes|.
-    3. Let |init| be a {{VideoFramePlaneInit}} with the following
-        keys:
-        1. Assign |timestamp| to {{VideoFrameInit/timestamp}}.
-        2. Assign |duration| to {{VideoFrameInit/duration}}.
-        3. Let {{VideoFramePlaneInit/codedWidth}} and
-            {{VideoFramePlaneInit/codedHeight}}
-            be the width and height of the decoded video frame |output| in
-            pixels, prior to any cropping or aspect ratio adjustments.
-        4. Let {{VideoFramePlaneInit/cropLeft}},
-            {{VideoFramePlaneInit/cropTop}},
-            {{VideoFramePlaneInit/cropWidth}}, and
-            {{VideoFramePlaneInit/cropHeight}}
-            be the crop region of the decoded video frame |output| in
-            pixels, prior to any aspect ratio adjustments.
-        5. Let {{VideoFramePlaneInit/displayWidth}} and
-            {{VideoFramePlaneInit/displayHeight}} be the display size of
-            the decoded video frame in pixels.
-    4. Return a new {{VideoFrame}}, constructed with |pixelFormat|,
-        |planes|, and |init|.
+        2. Let |displayAspectWidth| and |displayAspectHeight| be the
```

Comment:
Needs handling for when dispalyAspectWidth/Height are not provided

2. Let |displayAspectWidth| and |displayAspectHeight| be `undefined`.
3. If {{VideoDecoderConfig/displayAspectWidth}} and {{VideoDecoderConfig/displayAspectHeight}} [=map/exist=] in the {{VideoDecoder/[[active decoder config]]}}, assign their values to |displayAspectWidth| and |displayAsepctHeight| respectively.